### ISSUE TITLE
S. ratti assembly changed from BioProject PRJEA62033 to PRJEB125. Correct?

### ISSUE THREAD

**Issue by tharris** [WormBase/website#4081]
S. ratti has been broken for the entire development cycle. We just noticed it in the final stages of testing for WS249. Simple fix, just want to make sure that this change is correct.

**Comment by khowe** [comment]
Yes, that's correct. It switched in WS247. What's changed in WS249 is that it has been promoted to a core species.

**Comment by khowe** [comment]
...which means that it should be listed explicitly on the species drop-down (it's still in "Others" on staging.wormbase.org).

**Comment by tharris** [comment]
Ah I see. We autocreate config files based on species and bioproject but it does not extend to the actual database specifier. That should probably be fixed. So for the past two releases, S. ratti hasn't worked...

We'll take a look at the drop down during the extended development cycle for WS249.  Title edited to reflect.

**Comment by janelomax** [comment]
The species overview table needs to be updated for S. ratti as well, to Gene-set 'Curated'.

thanks!

**Comment by janelomax** [comment]
Note: S. ratti positioning in the drop-down is still broken in live WS249.

**Comment by sibyl229** [comment]
Hi @janelomax The links to S. ratti in species dropdown looks fine. Shall I close this?

**Comment by janelomax** [comment]
Hi @sibyl229 yes you can close this, thanks.
